### PR TITLE
Added recipe for FOSOAuthServerBundle

### PR DESCRIPTION
**Pull request by patrickbussmann**
| Q             | A
| ------------- | ---
| License       | MIT

Hopefully this can be merged soon to support FOSOAuthServerBundle 🙂

### CODE REVIEW

File: friendsofsymfony/oauth-server-bundle/1.6/config/packages/fos_oauth_server.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+fos_oauth_server:
+    db_driver:                  orm # Drivers available: orm, mongodb, or propel
+    client_class:               App\Entity\Client
+    access_token_class:         App\Entity\AccessToken
+    refresh_token_class:        App\Entity\RefreshToken
+    auth_code_class:            App\Entity\AuthCode
+#    model_manager_name:         ~ # change it to the name of your entity/document manager if you don't want to use the default one.
```
Comment (Nyholm) [inline friendsofsymfony/oauth-server-bundle/1.6/config/packages/fos_oauth_server.yaml]: We should not have config in comments like this. 
I would prefer a link to the docs instead.

File: friendsofsymfony/oauth-server-bundle/1.6/src/Entity/AccessToken.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ * @ORM\Entity(repositoryClass="App\Repository\AccessTokenRepository")
+ */
+class AccessToken extends BaseAccessToken
+{
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
+    protected $id;
```
Comment (Nyholm) [inline friendsofsymfony/oauth-server-bundle/1.6/src/Entity/AccessToken.php]: Protected? Is that really a good default?

File: friendsofsymfony/oauth-server-bundle/1.6/src/Repository/AccessTokenRepository.php [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    public function __construct(RegistryInterface $registry)
+    {
+        parent::__construct($registry, AccessToken::class);
+    }
+
+//    /**
+//     * @return AccessToken[] Returns an array of AccessToken objects
+//     */
+    /*
+    public function findByExampleField($value)
```
Comment (Nyholm) [inline friendsofsymfony/oauth-server-bundle/1.6/src/Repository/AccessTokenRepository.php]: I think this code in comments should be removed. 

Add a link to the docs instead.

File: friendsofsymfony/oauth-server-bundle/1.6/src/Repository/AuthCodeRepository.php [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+        parent::__construct($registry, AuthCode::class);
+    }
+
+//    /**
+//     * @return AuthCode[] Returns an array of AuthCode objects
+//     */
+    /*
+    public function findByExampleField($value)
+    {
+        return $this->createQueryBuilder('a')
```
Comment (Nyholm) [inline friendsofsymfony/oauth-server-bundle/1.6/src/Repository/AuthCodeRepository.php]: same here

**Comment by Nyholm** [comment]
Hey. Did you have time to fix my comments?

**Comment by Nyholm** [comment]
Closing due inactivity. 

#eu-fossa

**Comment by patrickbussmann** [comment]
No Sorry I have no time for fixing 😥

**Comment by Nyholm** [comment]
That is okey. Feel free to reopen it when you got time.

**Pull request closed without merging**